itest/collectible_split_test.go

```diff
@@ -0,0 +1,91 @@
+package itest
+
+import (
+	"context"
+
+	"github.com/lightninglabs/taro/tarorpc"
+	"github.com/stretchr/testify/require"
+)
+
+// testCollectibleSend tests that we can properly send a collectible asset
+// with split commitments.
+func testCollectibleSend(t *harnessTest) {
+	// First, we'll make a collectible with emission enabled.
+	rpcAssets := mintAssetsConfirmBatch(
+		t, t.tarod, []*tarorpc.MintAssetRequest{issuableAssets[1]},
+	)
+
+	familyKey := rpcAssets[0].AssetFamily.TweakedFamilyKey
+	genInfo := rpcAssets[0].AssetGenesis
+	genBootstrap := rpcAssets[0].AssetGenesis.GenesisBootstrapInfo
+
+	ctxb := context.Background()
+
+	// Now that we have the asset created, we'll make a new node that'll
+	// serve as the node which'll receive the assets.
+	secondTarod := setupTarodHarness(
+		t.t, t, t.lndHarness.BackendCfg, t.lndHarness.Bob, t.universeServer,
+	)
+	defer func() {
+		require.NoError(t.t, secondTarod.stop(true))
+	}()
+
+	// Next, we'll attempt to complete three transfers of the full value of
+	// the asset between our main node and Bob.
+	var (
+		numSends     = 3
+		fullAmount   = rpcAssets[0].Amount
+		receiverAddr *tarorpc.Addr
+		err          error
+	)
+
+	for i := 0; i < numSends; i++ {
```

Comment:
I think we can somewhat easily (factoring in the optional key family param) abstract this section into a sort of `pingPongAsset` helper func.